book/src/drivers.md

```diff
@@ -36,6 +36,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [Hammerspace CSI](https://github.com/hammer-space/csi-plugin) | `com.hammerspace.csi` | v0.3, v1.0 | A Container Storage Interface (CSI) Driver for Hammerspace Storage | Persistent | Read/Write Multiple Pods | Yes | Raw Block, Snapshot
 [Hitachi Vantara](https://knowledge.hitachivantara.com/Documents/Adapters_and_Drivers/Storage_Adapters_and_Drivers/Containers) | `com.hitachi.hspc.csi` | v1.0 | A Container Storage Interface (CSI) Driver for VSP series Storage | Persistent | ? | ? | 
 [HPE](https://github.com/hpe-storage/csi-driver) | `csi.hpe.com` | v0.3, v1.0, v1.1 | A Container Storage Interface (CSI) driver from HPE | Persistent or Ephemeral | Read/Write Single Pod | Yes | Raw Block, Snapshot, Expansion, Cloning
+[Huawei Storage CSI](https://github.com/Huawei/eSDK_K8S_Plugin) | `csi.huawei.com` | v1.0 | A Container Storage Interface (CSI) Driver for [Huawei Storage](https://support.huawei.com/enterprise/en/category/enterprise-storage-pid-1482606575525?submodel=21430818, https://support.huawei.com/enterprise/en/category/cloud-storage-pid-1482615290338?submodel=21430817) | Persistent | Read/Write Multiple Pod | Yes |
```

Comment:
Why are there two links for `Huawei Storage`? That breaks the markdown.

Reply:
Now there has only one link. Please check. Thank you!